proposals/proposal_1/lib/src/items/item_list.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:flutter/material.dart';
+import 'package:proposal_1/src/items/item.dart';
+import 'package:proposal_1/src/items/item_controller.dart';
+import 'package:proposal_1/src/items/item_details.dart';
+
+class ItemList extends StatefulWidget {
+  final ItemController controller;
+
+  const ItemList({Key key, @required this.controller}) : super(key: key);
+
+  @override
+  _ItemListState createState() => _ItemListState();
+}
+
+class _ItemListState extends State<ItemList> {
+  Item _selectedItem;
+
+  void _onItemTapped(Item item) {
+    setState(() {
+      _selectedItem = item;
+    });
+  }
+
+  @override
+  void initState() {
+    // Begin loading the users when the Item List is first shown.
+    widget.controller.initialize();
+
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return AnimatedBuilder(
+      animation: widget.controller,
+      builder: (context, _) {
+        if (widget.controller.loading) {
+          return const Center(child: CircularProgressIndicator());
+        } else if (widget.controller.error) {
+          return const Center(child: Text('Oh no! Could not load the users'));
+        }
+
+        return LayoutBuilder(builder: (context, constraints) {
+          return Scaffold(
+            appBar: AppBar(
+              title: const Text('Items'),
+              actions: [
+                IconButton(
+                  icon: const Icon(Icons.settings),
+                  onPressed: () {
+                    Navigator.pushNamed(context, '/settings');
+                  },
+                )
+              ],
+            ),
+            body: constraints.maxWidth < 600
+                ? _buildMobileContent(context)
+                : _buildTabletContent(context),
```

Comment:
```suggestion
                ? _buildNarrowContent(context)
                : _buildWideContent(context),
```

Maybe? We're not only supporting mobile devices.